### ISSUE TITLE
Problem installing...

### ISSUE THREAD

**Issue by shokru** [gragusa/lyapunov#1]
Dear Giuseppe,

my apologies for disturbing, but even after requesting devtools, I get this error message:
> devtools::install_github("gragusa/lyapunov")
Downloading GitHub repo gragusa/lyapunov@master
from URL https://api.github.com/repos/gragusa/lyapunov/zipball/master
Erreur : Could not find build tools necessary to build lyapunov

Would you know how to solve it?
Many thanks,

Guillaume

**Comment by gragusa** [comment]
I think you need a FORTRAN compiler installed. If you are using ubuntu install the build-essential package. If you are on windows as I suspect you have to have a tool chain installed.

**Comment by shokru** [comment]
Dear Giuseppe,

thank you for your reply. I'll will try & see.
Regards,

GC

**Comment by MaxMillerLab** [comment]
Giuseppe,

Thanks for posting! I'm having some trouble installing. I have all the necessary tools installed, but I'm getting this error,

<img width="688" alt="screen shot 2018-03-27 at 3 16 34 pm" src="https://user-images.githubusercontent.com/37848132/37989753-4f47ed92-31d2-11e8-8894-a177b967bc18.png">

Any ideas with how to move past this? Thank you for your time!

Max

**Comment by gragusa** [comment]
Which fortran are you using? I am guessing from the errors, you are on OSX. Is it correct? Which version of OSX?

**Comment by MaxMillerLab** [comment]
Good catch! I'm on OSX-High Sierra, the newest Mac update. And I have GNU Fortran (GCC) 4.2.3. Thanks for your help!

**Comment by gragusa** [comment]
I think you need a newer gfortran.  (R 3.4.0 El Capitan binaries are using Clang 4.0.0 and GNU Fortran 6.1 to provide OpenMP parallelization support and C++17 standard features.) For instance, on my machine I have 

```
GNU Fortran (Homebrew GCC 7.3.0_1) 7.3.0
Copyright (C) 2017 Free Software Foundation, Inc.
This is free software; see the source for copying conditions.  There is NO
warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
```


```
R 3.4.0 El Capitan binaries are using Clang 4.0.0 and GNU Fortran 6.1 to provide OpenMP parallelization support and C++17 standard features. If you want to compile R packages from sources, please download GNU Fortran binary from the official GNU Fortran Binaries page - in particular OS X 10.11 gfortran 6.1. We are also providing Clang 4.0.0 binaries for OS X 10.11 and higher in our libs directory (note that the offical Clang 4.0.0 binaries only support macOS 10.12) and will provide an Apple Installer package here soon.
```
